Salesforce_Advanced/Merging_Duplicate_Records/LeadDedupe.xml

```diff
@@ -0,0 +1,581 @@
+			</groupby>
+			<havings>
+				<having field="Id" fieldfunction="COUNT" operator="gt" value="2" enclosevalueinquotes="false"/>
+			</havings>
+		</model>
+		<model id="Master_Lead" limit="1" query="false" createrowifnonefound="false" sobject="Lead" doclone="" type="" datasource="salesforce" orderby="CreatedDate ASC">
+			<fields>
+				<field id="Name"/>
+				<field id="Email"/>
+				<field id="Id"/>
+				<field id="Company"/>
+				<field id="LeadSource"/>
+				<field id="Phone"/>
+				<field id="Address"/>
+				<field id="Country"/>
+				<field id="Status"/>
+				<field id="OwnerId"/>
+				<field id="Owner.Name"/>
+				<field id="CreatedDate"/>
+				<field id="City"/>
+				<field id="Street"/>
+				<field id="State"/>
+				<field id="PostalCode"/>
+				<field id="Description"/>
+			</fields>
+			<conditions>
+				<condition type="fieldvalue" value="false" enclosevalueinquotes="false" field="IsConverted"/>
+				<condition type="fieldvalue" value="" field="Name" operator="=" mergefield="Name" novaluebehavior="noquery" state="filterableoff" inactive="true" name="Name" enclosevalueinquotes="true" clientorserver="server"/>
+				<condition type="fieldvalue" value="" field="Email" operator="=" mergefield="Email" novaluebehavior="noquery" state="filterableoff" inactive="true" name="Email" enclosevalueinquotes="true" clientorserver="server"/>
+			</conditions>
+			<actions/>
+		</model>
+		<model id="Dupe_Leads_List" limit="20" query="false" createrowifnonefound="false" sobject="Lead" doclone="" type="" datasource="salesforce">
+			<fields>
+				<field id="Name"/>
+				<field id="Email"/>
+				<field id="Id"/>
+				<field id="Company"/>
+				<field id="LeadSource"/>
+				<field id="Phone"/>
+				<field id="Address"/>
+				<field id="Country"/>
+				<field id="Status"/>
+				<field id="OwnerId"/>
+				<field id="Owner.Name"/>
+				<field id="City"/>
+				<field id="State"/>
+				<field id="Description"/>
+				<field id="Street"/>
+				<field id="PostalCode"/>
+			</fields>
+			<conditions>
+				<condition type="fieldvalue" value="false" enclosevalueinquotes="false" field="IsConverted"/>
+				<condition type="fieldvalue" value="" field="Name" operator="=" mergefield="Name" novaluebehavior="noquery" state="filterableoff" inactive="true" name="Name" enclosevalueinquotes="true" clientorserver="server"/>
+				<condition type="fieldvalue" value="" field="Email" operator="=" mergefield="Email" novaluebehavior="noquery" state="filterableoff" inactive="true" name="Email" enclosevalueinquotes="true" clientorserver="server"/>
+				<condition type="modelmerge" value="" field="Id" operator="!=" model="Master_Lead" enclosevalueinquotes="true" mergefield="Id" novaluebehavior="noquery"/>
+			</conditions>
+			<actions/>
+		</model>
+		<model id="ReparentActivities" limit="20" query="false" createrowifnonefound="false" datasource="salesforce" sobject="Task">
+			<fields>
+				<field id="RecordTypeId"/>
+				<field id="RecordType.Name"/>
```

Comment:
Will cause errors in orgs without Record Type enabled, and they aren't needed for this page.
```suggestion
```